bin/BuildPackages.sh

```diff
@@ -190,6 +191,10 @@ build_one_package() {
   (  # start subshell
   set -e
   cd "$CURDIR/$PKG"
+  local PKG_NAME=$(../../bin/gap.sh -q -T <<< 'Read("../../bin/BuildPackagesHelper.g");')
```

Comment:
Should find the gap executable at $GAPROOT. Also, could just use $GAPROOT/gap I think